app/controllers/spree/admin/tax_rates_controller.rb

```diff
@@ -30,7 +30,7 @@ def amount_changed?
       end
 
       def associated_adjustments?
-        Spree::Adjustment.where(originator: @tax_rate).exists?
+        Spree::Adjustment.exists?(originator: @tax_rate)
```

Comment:
This version is definitely shorter. But I like the old version more. Can I justify it?

_Habit._ Our code base has 8 violations of this style but only 4 violations of the inverse style of preferring conditions in the where clause.

_Fewer tools._ Ruby on Rails allows you to achieve the same thing in different ways. While that can be very convenient, it's also a burden. Keeping many different ways to do something (many different tools) means that you need to remember all of them and what they mean. Code becomes more difficult to read when you read two different things and have to think: "Is that doing the same?" That's why Rubocop enforces certain styles. Consistency improves readability and mental load. In this example, `where` is the standard tool to deal with conditions and `exists?` can just be added on without having any parameters (or me knowing that it could have parameters). It's simple. But `exists?` does have a rich API which we currently barely use (4 uses in the whole code base). Using `exists?` for the same purpose we would usually use `where` for just introduces another tool we don't need.

_Open to change._ Our code changes all the time. We may have an association with certain `where` conditions and we print the whole list on the screen. But then the requirement changes and we just want the number of records. And then we just want to know if there are any records at all. Instead of rewriting the `where`, we can just add `count` or `exists?` at the end. I think that it makes code more re-usable when using `where` and don't add conditions to `exists?`.

Reply:
Thanks both. 
I agree with Maikel's reasoning, although it's slightly more code I find it much clearer that way.

Therefore, Cyrille could you please change the Rubocop config to "EnforcedStyle: where" and update accordingly?

Reply:
( I forgot to press send on my answer earlier :man_facepalming: ) 
I agree with Maikel as well, it doesn't really improve the code, in my opinion it actually decrease the readability by hiding the `where`.

Reply:
Hello @dacook , I have kept the rubocop_todo modifications and added an exclusion in the `.rubocop_styleguide.yml` file. So no offenses will be raised for this cop.

Reply:
David's suggestion was to change the cop to `EnforcedStyle: where`. Instead you disabled it. Either is okay with me. I think I would have preferred the enforced style.

@cyrillefr, does that mean that you have a strong preference for the exists syntax and want the style rules to be open for that?

I'm open to just disabling it as well. It's really not that important.